docs/adr/0012-high-level-design-infra-cluster.md

```diff
@@ -0,0 +1,119 @@
+## ABOUT THIS DOCUMENT
+
+This document is a High-level Design for the new MOC infra management cluster. This hub cluster will be the central openshift cluster to manage other clusters on-demand using infrastructure as a code to deploy configuration. This centralized cluster will be part of Operate First with RedHat products, testing new dev preview capabilities and providing feedback.
+
+This document will be publicly shared for the open-source community for others to learn. Providing a full technical stack overview that can be automated and step up in one-click operation.
+
+
+## AUDIENCE
+
+The intended audience for this document is any person with interest in deploying a similar solution as described in this document.
+
+## REVISIONS
+
+
+| Name | Version | Date | Comments |
+|------| --------| -----| ---------|
+| Ilana Polonsky | 1.0 | 02/16/2021 | MOC infra leader |
+| Oren Oichman | 1.1 | 02/16/2021 | Senior Cloud Consultant |
+
+## CURRENT STATUS
+
+Our current status consisted of three separate OpenShift clusters one for running development tasks, one for Infrastructure testing and a single (all in one) node cluster.
+
+### Details about the current clusters
+
+*   Lars’s cluster - A 6 nodes openshift cluster that runs Argocd to deploy application and configuration.
```

Comment:
The cluster has *9* nodes. Probably worth noting that it is running OCS for storage.